man/systemd.resource-control.xml

```diff
@@ -913,6 +913,38 @@ DeviceAllow=/dev/loop-control
           </para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>ManagedOOMPreference=none|avoid|omit</varname></term>
+
+        <listitem>
+          <para>Allows deprioritizing or omitting this unit's cgroup as a candidate when <command>systemd-oomd</command>
+          needs to act. Requires support for extended attributes (see
+          <citerefentry project='man-pages'><refentrytitle>xattr</refentrytitle><manvolnum>7</manvolnum></citerefentry>)
+          in order to use <option>avoid</option> or <option>omit</option>. Additionally, <command>systemd-oomd</command>
+          will ignore these extended attributes if the unit's cgroup is not owned by the root user and group.</para>
```

Comment:
hmm, i wouldn't mention the implementation detail that is the xattr stuff. how oomd and PID 1 communicate should not be relevant for users, no? and should be something we could theoretically change any time, right?

Reply:
i left this detail in since the avoid/omit feature is useful outside of the systemd service manager (and outside of cgroups managed by systemd). for example, internally we have container tasks set up similar to the following:
```
/workload.slice (ManagedOOMMemoryPressure=kill)
    /container1.service (Delegate=yes)
        /logger
        /task_root
```
the container manager can then set the avoid/omit xattr on `/logger` to encourage oomd to target `/task_root` first.